staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/generators/applyconfiguration.go

```diff
@@ -252,20 +253,39 @@ func (g *applyConfigurationGenerator) generateMemberWith(sw *generator.SnippetWr
 	sw.Do("}\n", memberParams)
 }
 
-func (g *applyConfigurationGenerator) generateMemberWithForSlice(sw *generator.SnippetWriter, memberParams memberParams) {
+func (g *applyConfigurationGenerator) generateMemberWithForSlice(sw *generator.SnippetWriter, member types.Member, memberParams memberParams) {
+	memberIsPointerToSlice := member.Type.Kind == types.Pointer
```

Comment:
I don't know what the current test approach for this generator is, but is there any way to exercise this with a test? if we're not using this in-tree, we won't know if this works or if a future change broke someone depending on this out of tree

Reply:
> I don't know what the current test approach for this generator is,

The current test approach appears to be "generate the code and see if it works in our other tests". It's not perfect, but given that openshift pulls in kube betas to vendor and it now generates these types, the drift and impact is limited.